builtins/amp-video.js

```diff
@@ -45,11 +52,20 @@ export function installVideo(win) {
             'No "poster" attribute has been provided for amp-video.');
       }
 
+      videoManagerForDoc(this.win.document).register(this);
```

Comment:
What would be the alternative to this. Querying DOM elements for specific tags (`amp-video` for now) when the video manager is installed and registering them then?